server/controller/questions.js

```diff
@@ -2,9 +2,10 @@ const getQuestions = require('../database/query/questions');
 const { questions } = require('../database/models');
 
 exports.get = async (request, response) => {
+  const userId = request.id;
```

Comment:
@lubnaabd This works good since there's not really any private data for that user being returned by this service (it only uses the userId to filter the questions by dialect).  If there were some private data, the best practice would be to get the userId from the authentication token instead of the request URL, to guarantee that only the authenticated user can request data for themselves.

Reply:
@tomduggan85  actually  ``` request.id; ``` is from the authentication token.

Reply:
Oh, sorry 😂 nevermind me!